aquadoggo/src/replication/message.rs

```diff
@@ -13,32 +13,37 @@ use serde::{Deserialize, Serialize};
 
 use crate::replication::{Mode, SessionId, TargetSet};
 
-pub const SYNC_REQUEST_TYPE: MessageType = 0;
-pub const ENTRY_TYPE: MessageType = 8;
-pub const SYNC_DONE_TYPE: MessageType = 9;
+pub const ANNOUNCE_TYPE: MessageType = 0;
+pub const SYNC_REQUEST_TYPE: MessageType = 1;
+pub const SYNC_DONE_TYPE: MessageType = 2;
+pub const ENTRY_TYPE: MessageType = 3;
```

Comment:
I've changed the identifiers here, not sure if this is how we want to do it but in the specification we didn't set an identifier for "Announce" messages.